app/src/main/java/deakin/gopher/guardian/view/patient/PatientAddFragment.java

```diff
@@ -10,188 +10,188 @@
 import android.view.ViewGroup;
 import android.widget.Button;
 import android.widget.EditText;
+
 import androidx.annotation.NonNull;
 import androidx.annotation.RequiresApi;
 import androidx.fragment.app.Fragment;
 import androidx.viewpager2.widget.ViewPager2;
+
 import deakin.gopher.guardian.R;
 import deakin.gopher.guardian.model.Patient;
 import deakin.gopher.guardian.util.DataListener;
 
 public class PatientAddFragment extends Fragment {
-  // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
```

Comment:
Same comment as before with this giant formatting change. Doesn't seem like it belongs in this PR